package.json

```diff
@@ -620,6 +620,100 @@
         "title": "Open help for path",
         "category": "R",
         "command": "r.helpPanel.openForPath"
+      },
+      {
+        "title": "Toggle Style",
+        "category": "R Httpgd",
```

Comment:
I guess "R Plot" might be a better name for the category? `httpgd` is the package name and we rely on the package as the backend to provide R plot in vscode. For end users, they might only need to think of "plot" instead of the specific package we rely on?

Reply:
Makes sense. For now I wanted to make sure that users know what's part of the httpgd-based viewer and what's part of the "old" viewer, but if the old viewer does not have any settings/commands that might lead to confusion, we can also rename as you suggested.

Reply:
The old plot does not have any setting in vscode but an option in R `vsc.plot` to control which view column to show the png plot file or `FALSE` to disable.

I guess we could rename vscode settings from `r.httpgd.*` to `r.plot.*` without producing any confusion?

Reply:
Sounds good